lib/index.js

```diff
@@ -10,7 +10,7 @@ var internals = {
 internals.store.lookUp = function (obj, method) {
   var i = this.length;
   var lookup;
-  while (i-- && !lookup) {
+  while (!lookup && i--) {
```

Comment:
It's more readable than maintaining another variable and having a more complex check on every iteration of the loop.